### PR TITLE
docs: updates for standalone operator env vars

### PR DESCRIPTION
**Pull request by PaulRMellor**
Signed-off-by: prmellor <pmellor@redhat.com>

**Documentation**

- Updates the standalone operator deployment instructions in the _Deploying_ guide to clarify how the _labels_ environment variables should be used. 
- Notes the requirement for configuring the environment variables in the operator sections of the _Configuring_ guide.

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Write tests
- [ ] Make sure all tests pass
- [x] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md
- [ ] Supply screenshots for visual changes, such as Grafana dashboards

### CODE REVIEW

File: documentation/assemblies/deploying/assembly-deploy-create-cluster.adoc [hunk 1]
```diff
@@ -17,8 +17,7 @@ But you can deploy and use the Topic Operator and User Operator as standalone co
 NOTE: The Cluster Operator can watch one, multiple, or all namespaces in a Kubernetes cluster.
 The Topic Operator and User Operator watch for `KafkaTopics` and `KafkaUsers` in the single namespace of the Kafka cluster deployment.
 
-[discrete]
-== Deploying a Kafka cluster with the Topic Operator and User Operator
+.Deploying a Kafka cluster with the Topic Operator and User Operator
```
Comment (scholzj) [inline documentation/assemblies/deploying/assembly-deploy-create-cluster.adoc:20]: Will this keep these titles linkable? I think we really want to be able to deep link to them.

File: documentation/assemblies/deploying/assembly-deploy-create-cluster.adoc [hunk 2]
```diff
@@ -28,8 +27,7 @@ Perform these deployment steps if you want to use the Topic Operator and User Op
 .. xref:deploying-the-topic-operator-using-the-cluster-operator-{context}[Topic Operator]
 .. xref:deploying-the-user-operator-using-the-cluster-operator-{context}[User Operator]
 
-[discrete]
-== Deploying a standalone Topic Operator and User Operator
+.Deploying a standalone Topic Operator and User Operator
```
Comment (scholzj) [inline documentation/assemblies/deploying/assembly-deploy-create-cluster.adoc:30]: Same as above.

File: documentation/assemblies/operators/assembly-using-the-topic-operator.adoc [hunk 3]
```diff
@@ -5,13 +5,27 @@
...
+
+For deployment instructions, see the following:
+
+* link:{BookURLDeploying}#deploying-the-topic-operator-using-the-cluster-operator-{context}[Deploying the Topic Operator using the Cluster Operator (recommended)^]
+* link:{BookURLDeploying}#deploying-the-topic-operator-standalone-{context}[Deploying the standalone Topic Operator^]
+
+[IMPORTANT]
+====
+To deploy the standalone Topic Operator, you need to set environment variables to connect to a Kafka cluster.
+These environment variables are internal to Strimzi and do not need to be set if you are deploying the Topic Operator using the Cluster Operator.
```
Comment (scholzj) [inline documentation/assemblies/operators/assembly-using-the-topic-operator.adoc:27]:
```suggestion
These environment variables do not need to be set if you are deploying the Topic Operator using the Cluster Operator as they will be set by the Cluster Operator.
```

File: documentation/assemblies/operators/assembly-using-the-user-operator.adoc [hunk 4]
```diff
@@ -5,15 +5,27 @@
...
+
+For deployment instructions, see the following:
+
+* link:{BookURLDeploying}#deploying-the-user-operator-using-the-cluster-operator-{context}[Deploying the User Operator using the Cluster Operator (recommended)^]
+* link:{BookURLDeploying}#deploying-the-user-operator-standalone-{context}[Deploying the standalone User Operator^]
+
+[IMPORTANT]
+====
+To deploy the standalone User Operator, you need to set environment variables to connect to a Kafka cluster.
+These environment variables are internal to Strimzi and do not need to be set if you are deploying the User Operator using the Cluster Operator.
```
Comment (scholzj) [inline documentation/assemblies/operators/assembly-using-the-user-operator.adoc:27]: Same as above

File: documentation/modules/deploying/proc-deploy-topic-operator-standalone.adoc [hunk 5]
```diff
@@ -87,7 +87,11 @@ spec:
 <1> The Kubernetes namespace for the Topic Operator to watch for `KafkaTopic` resources. Specify the namespace of the Kafka cluster.
 <2> The host and port pair of the bootstrap broker address to discover and connect to all brokers in the Kafka cluster.
 Use a comma-separated list to specify two or three broker addresses in case a server is down.
-<3> The label selector to identify the `KafkaTopic` resources managed by the Topic Operator.
+<3> The label to identify the name of the `KafkaTopic` resources managed by the Topic Operator.
```
Comment (scholzj) [inline documentation/modules/deploying/proc-deploy-topic-operator-standalone.adoc:90]:
```suggestion
<3> The label to identify the `KafkaTopic` resources managed by the Topic Operator.
```

File: documentation/modules/deploying/proc-deploy-user-operator-standalone.adoc [hunk 6]
```diff
@@ -83,7 +83,11 @@ spec:
 Use a comma-separated list to specify two or three broker addresses in case a server is down.
 <3> The Kubernetes `Secret` that contains the public key (`ca.crt`) value of the Certificate Authority that signs new user certificates for TLS client authentication.
 <4> The Kubernetes `Secret` that contains the private key (`ca.key`) value of the Certificate Authority that signs new user certificates for TLS client authentication.
-<5> The label selector used to identify the `KafkaUser` resources managed by the User Operator.
+<5> The label to identify the name of the `KafkaUser` resources managed by the User Operator.
```
Comment (scholzj) [inline documentation/modules/deploying/proc-deploy-user-operator-standalone.adoc:86]: Same as above.

File: documentation/assemblies/deploying/assembly-deploy-create-cluster.adoc (hunk 1 above)
Comment (PaulRMellor) [reply documentation/assemblies/deploying/assembly-deploy-create-cluster.adoc:20]: They weren't linkable with the [discrete]. If you think that's important, I've set as level three headings without discrete.

**Review by scholzj — APPROVED** [review]

**Review by ppatierno — APPROVED** [review]

**Pull request merged**